addons/crm/models/crm_lead.py

```diff
@@ -1103,6 +1103,9 @@ def _notify_get_groups(self, message, groups):
                 {'url': won_action, 'title': _('Won')},
                 {'url': lost_action, 'title': _('Lost')}]
 
+        if self.team_id:
+            salesman_actions.append({'url': '/mail/view?model=crm.team&res_id=%s' % (self.team_id.id), 'title': _('Sales Team Settings')})
```

Comment:
You should use _notify_get_action_link. Purpose is to avoid hardcoding links in various modules so that if we have changes to do we have to do it once.